src/server.cpp

```diff
@@ -1881,6 +1881,20 @@ void Server::SendSetSky(u16 peer_id, const video::SColor &bgcolor,
 	Send(&pkt);
 }
 
+void Server::SendCloudsParams(u16 peer_id, const float density,
+		const video::SColor &color_bright,
+		const video::SColor &color_ambient,
+		const float height,
+		const float thickness,
+		const v2f speed)
+{
+	NetworkPacket pkt(TOCLIENT_CLOUDS_PARAMS, 0, peer_id);
+	pkt << (u16) (density  * 65535) << color_bright << color_ambient
```

Comment:
True. I was trying to save 2 bytes there, but with all those colors I reckon it does not matter anymore :-P